.github/workflows/nightly-examples.yml

```
@@ -0,0 +1,59 @@
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: ".python-version"
+          cache: ${{ env.CACHE_TYPE }}
+
+      - name: Install dependencies
+        run: uv sync --dev --all-extras
+
+      - name: Install patchright
+        if: steps.cache-patchright.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache-patchright" is not defined in object type {}

(expression)